src/components/loadflow-result.js

```diff
@@ -117,144 +118,186 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
+                }),
+                field: 'synchronousComponentNum',
+            },
+            {
+                headerName: intl.formatMessage({ id: 'status' }),
+                field: 'status',
+                cellRenderer: StatusCellRender,
+            },
+            {
+                headerName: intl.formatMessage({
+                    id: 'iterationCount',
+                }),
+                field: 'iterationCount',
+            },
+            {
+                headerName: intl.formatMessage({
+                    id: 'slackBusId',
+                }),
+                field: 'slackBusId',
+            },
+            {
+                headerName: intl.formatMessage({
+                    id: 'slackBusActivePowerMismatch',
+                }),
+                field: 'slackBusActivePowerMismatch',
+                cellRenderer: NumberRenderer,
+            },
+        ];
+    }, [intl, NumberRenderer, StatusCellRender]);
 
     function renderLoadFlowResult() {
         return (
             <Paper className={classes.tablePaper}>
-                <VirtualizedTable
-                    rows={result.componentResults}
-                    sortable={true}
-                    columns={[
-                        {
-                            label: intl.formatMessage({
-                                id: 'connectedComponentNum',
-                            }),
-                            dataKey: 'connectedComponentNum',
-                        },
-                        {
-                            label: intl.formatMessage({
-                                id: 'synchronousComponentNum',
-                            }),
-                            dataKey: 'synchronousComponentNum',
-                        },
-                        {
-                            label: intl.formatMessage({
-                                id: 'status',
-                            }),
-                            dataKey: 'status',
-                            cellRenderer: StatusCellRender,
-                        },
-                        {
-                            label: intl.formatMessage({
-                                id: 'iterationCount',
-                            }),
-                            dataKey: 'iterationCount',
-                        },
-                        {
-                            label: intl.formatMessage({
-                                id: 'slackBusId',
-                            }),
-                            dataKey: 'slackBusId',
-                        },
-                        {
-                            label: intl.formatMessage({
-                                id: 'slackBusActivePowerMismatch',
-                            }),
-                            dataKey: 'slackBusActivePowerMismatch',
-                            cellRenderer: NumberRenderer,
-                        },
-                    ]}
+                <CustomAGGrid
```

Comment:
I reckon you should add `enableCellTextSelection={true}` since cells values are not selectable by default and the previous implementation allowed for selection

Reply:
I added it.